app/controllers/admin/domains/registry_lock_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Admin
+  module Domains
+    class RegistryLockController < BaseController
+
+      def destroy
+        set_domain
+        authorize! :manage, @domain
+        if @domain.remove_registry_lock
+          redirect_to edit_admin_domain_url(@domain), notice: t('admin.domains.registry_lock_delete.success')
```

Comment:
Why not just `notice: t('.deleted')`?

Reply:
Because `Deleted` does not fit in this scenario. It certainly fits less than `Registry lock removed`, as we are not deleting a record, we are removing a restriction.

Reply:
I mean why do you need to spell out the full translation path, instead of using a shortcut?